EVASEAC/app/screens/HomeScreen.tsx

```diff
@@ -1,32 +1,242 @@
+
+            <View style={styles.fast_access}>
+                <Text style={styles.text}> Accessos rápidos </Text>
+                <View style={styles.fast_access_buttons}>
+                    <View style={{ alignItems: "center" }}>
+                        <Pressable style={styles.circular_button} onPress={() => navigateTo("home")}>
+                            <FontAwesomeIcon name="map" size={30} color="#ffff" />
+                        </Pressable>
+                        <Text style={{ fontSize: 12 }}> Nuevo sitio </Text>
+                    </View>
+
+                    <View style={{ alignItems: "center" }}>
+                        <Pressable style={styles.circular_button} onPress={() => navigateTo("home")}>
+                            <FontAwesomeIcon name="history" size={30} color="#ffff" />
+                        </Pressable>
+                        <Text style={{ fontSize: 12 }}> Historial </Text>
+                    </View>
+
+                    <View style={{ alignItems: "center" }}>
+                        <Pressable style={styles.circular_button} onPress={() => navigateTo("Protocol1")}>
+                            <Text style={styles.button_text}> 1 </Text>
+                        </Pressable>
+                        <Text style={{ fontSize: 12 }}> Protocolo 1 </Text>
+                    </View>
+
+                    <View style={{ alignItems: "center" }}>
+                        <Pressable style={styles.circular_button} onPress={() => navigateTo("Protocol2")}>
+                            <Text style={styles.button_text}> 2 </Text>
+                        </Pressable>
+                        <Text style={{ fontSize: 12 }}> Protocolo 2 </Text>
+                    </View>
+                </View>
+            </View>
+
+            <View style={styles.last_action}>
+                <Text style={styles.text}>Última acción / En proceso</Text>
+                <View style={styles.box}>
+                    <Text style={styles.box_text}>Protocolo 3</Text>
+                </View>
+            </View>
+
+            <View style={styles.last_action}>
+                <Text style={styles.text}>Último sitio / En proceso</Text>
+                <ImageBackground
+                    style={styles.box}
+                    imageStyle={styles.image_background}
+                    source={require("../../assets/map.png")}
+                >
+                    <Text style={styles.box_text}>ESCOM</Text>
+                </ImageBackground>
+            </View>
+
+            <View style={{ flex: 0.3, alignItems: 'center', alignContent: 'center' }}>
+                <TouchableOpacity onPress={toggleDropdown}>
+                    <View style={styles.button}>
+                        <Text style={[{ fontSize: 35, }, styles.box_text]}>Más opciones</Text>
+                    </View>
+                </TouchableOpacity>
+            </View>
+
+            <Modal style={styles.container} visible={dropdownVisible}>
```

Comment:
Maybe we could also make more modules to have shorter and more modularized codes. We should think about directories structures for each modules [in the future](https://github.com/orgs/evaseac/projects/2/views/1)